xbmc/pvr/guilib/PVRGUIActions.cpp

```diff
@@ -1913,10 +1913,7 @@ namespace PVR
       bResetClients = selector.IsResetClientsSelected();
     }
 
-    CDateTime::ResetTimezoneBias();
-
-    CLog::LogFC(LOGDEBUG, LOGPVR, "PVR clearing {} database",
-                bResetEPGOnly ? "EPG" : "PVR and EPG");
+    CLog::LogFC(LOGDEBUG, LOGPVR, "PVR clearing %s database", bResetEPGOnly ? "EPG" : "PVR and EPG");
 
```

Comment:
GGoing t

Reply:
Yeah, let me check once more - I have numerous swings with these already

Reply:
Fixed and verified no other stuff like this happens across the rest of patchset